mlir/include/mlir/Dialect/Rock/IR/RockAttrDefs.td

```diff
@@ -243,11 +247,14 @@ def Rock_XdlopsGemmParamsAttr : Rock_Attr<"XdlopsGemmParams", [RockTuningParamAt
 
   let extraClassDeclaration = [{
     std::string getPerfConfigStr(){
-      const Twine paramsConcat = Twine(getMPerBlock()) + ","
+      const Twine paramsConcat = 
+          Twine(getMPerBlock()) + ","
         + Twine(getNPerBlock()) + ","
         + Twine(getKPerBlock()) + ","
+        + Twine(getKpack()) + ","
         + Twine(getMPerWave()) + ","
-        + Twine(getNPerWave()) + ",1,1";
+        + Twine(getNPerWave()) + ","
+        + "1,1"; /* *ThreadCopyMore* */
```

Comment:
Same with previous comment: Adjust the sequence to make them compatible.

https://github.com/ROCmSoftwarePlatform/rocMLIR/blob/7898c00ad5c4aae24add7c276123ba3be01c6eff/mlir/include/mlir/Dialect/Rock/Tuning/GridwiseGemmParams.h#L88-L96

Reply:
Please fix position of kPack